sprint_tutorial/tests/test_compute.py

```diff
@@ -1,13 +1,20 @@
 """ Tests for the compute module
 """
-from nose.tools import assert_equal
 
-from sprint_tutorial.compute import my_sum
+from sprint_tutorial.compute import my_sum, my_power
+import pytest
 
-#run nosetests
+def test_my_sum():
+    assert my_sum(0, 0) == 0
+    assert my_sum(3, 0) == 3
+    assert my_sum(99, 1) == 100
 
-def test_my_sum1():
-    assert_equal(my_sum(3, 5), 8)
 
-def test_my_sum2():
-    assert_equal(my_sum(5, 5), 10)
+def test_my_pow():
+    assert my_power(2, 4) == 16
+    assert my_power(5, 2) == 25
+
```

Comment:
Hey congrats on the PR! My feedback then: only 1 blank line between the 2 functions isn't pep8...